<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>271</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="expTypeLabel">
       <property name="font">
        <font>
         <family>나눔스퀘어_ac</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>실험유형</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="expTypeCBox">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>pre_A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pre_B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>post_A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>post_B</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="font">
        <font>
         <family>나눔스퀘어_ac</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>피험자명</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="expCntLEdit">
       <property name="font">
        <font>
         <family>나눔스퀘어_ac</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameLEdit">
       <property name="font">
        <font>
         <family>나눔스퀘어_ac</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>InTaekJung</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="expCntLabel">
       <property name="font">
        <font>
         <family>나눔스퀘어_ac</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>실험회차</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="startBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>나눔스퀘어_ac</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>문제풀이 시작</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>440</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어_ac</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>[시선응답] Lecture &amp; Test</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
